Author: Vicentiu Ciorbaru <vicentiu@mariadb.org>
Date:   Wed Jul 4 13:29:55 2018 +0300

    Disable disks.disks test

    This failed on most Launchpad architectures, as the disk information
    does not seem to be available on Launchpad builders. A better fix will
    be done by upstream later to the actual test code itself.

--- /dev/null
+++ b/plugin/disks/mysql-test/disks/disabled.def
@@ -0,0 +1 @@
+disks                    : does not work on Launchpad
